multichain-testing/scripts/noble-update-genesis.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/bash
```

Comment:
with all the `jq` machinations, is bash really easier than .js (with zx / execa) for this sort of thing?

Maybe bash is conventional among our peers that do chain node ops, though.

Reply:
Good point! Starship expects a shell script but it needed be bash. It could even be TypeScript with,
```sh
#!/usr/bin/env -S node --import ts-blank-space/register
```